Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -407,6 +407,24 @@ class ExportBrowserDataSetting: HiddenSetting {
     }
 }
 
+class EnableActivtyStreamSetting: HiddenSetting {
+    let profile: Profile
+
+    override init(settings: SettingsTableViewController) {
+        self.profile = settings.profile
+        super.init(settings: settings)
+    }
+
+    override var title: NSAttributedString? {
+        // Not localized for now.
+        return NSAttributedString(string: "Enable the New Tab Experience", attributes: [NSForegroundColorAttributeName: UIConstants.TableViewRowTextColor])
+    }
+
+    override func onClick(_ navigationController: UINavigationController?) {
+        FeatureSwitches.activityStream.enrollMember(profile.prefs)
```

Comment:
Hmm. In the case where 100% of the cohort get the new feature, then I guess we'll remove the setting?

The feature switch is about early access to a new feature, rather than giving users ultimate choice about features. They get to choose if they are involved in the early access. 

I'm slightly queasy about giving users an irreversible choice without even a confirmation dialog.